qemu.robot

```diff
@@ -16,8 +16,8 @@ Preparing test environment
   Set Suite Variable    ${application_name}    %{application_name}
   Should Not Be Empty   ${application_name}    msg=application_name variable cannot be blank
   Set Suite Variable    ${device_type}    %{device_type}
-  Set Suite Variable    ${RESINRC_RESIN_URL}    %{RESINRC_RESIN_URL}
-  Set Suite Variable    ${RESINRC_PROXY_URL}    %{RESINRC_PROXY_URL}
+  Set Suite Variable    ${BALENARC_BALENA_URL}    %{BALENARC_BALENA_URL}
+  Set Suite Variable    ${BALENARC_PROXY_URL}    %{BALENARC_PROXY_URL}
   Set Suite Variable    ${image}    %{image}
   File Should Exist     ${image}  msg="Provided images file does not exist"
   Set Suite Variable    ${application_repo}    https://github.com/resin-io/autohat-ondevice.git
```

Comment:
Does this URL also need updating? :thinking:  It currently redirects to https://github.com/balena-io/autohat-ondevice

Reply:
It's actually a critical piece of this stack of cards - don't ask me how I know :)